lib/smart_proxy_pulp_plugin/pulp_api.rb

```diff
@@ -17,5 +17,12 @@ class Api < Sinatra::Base
         log_halt 503, "Pulp server '#{URI.parse(::PulpProxy::Plugin.settings.pulp_url.to_s).host}' is unknown"
       end
     end
+
+    get "/status/diskfree" do
+      disk_raw_data = `df -h #{PulpProxy::Plugin.settings.pulp_dir}`.split("\n").last
```

Comment:
So, I was thinking to do that, but since it is really a one-liner no sure we need?
Also, testing the API here is much like testing the class :)